README.md

```diff
@@ -17,6 +17,7 @@ Available addons
 ----------------
 addon | version | summary
 --- | --- | ---
+[auditlog](auditlog/) | 10.0.1.0.0 | Audit Log
```

Comment:
Please revert this change - README.md is managed by a bot, so this change will create some conflicts.